Spatial.gd

```diff
@@ -922,6 +1039,11 @@ func _on_Settings_pressed():
 func _on_MarkersUI_cell_selected():
 	var category_item = self.markers_ui.get_selected()
 	self.currently_active_category = category_item
+	self.currently_active_path2d = null
+	self.currently_active_path3d = null
+	self.currently_active_waypoint_trail = null
+	on_gizmo_deselected()
+	clear_adjustment_nodes()
```

Comment:
What I found in my testing is that if you have the gizmos open and select a new category, the gizmos stay. Since the current flow is to have the user select a category before the gizmos are made, I think it logically follows that they should only be shown when that category is still selected. I think for now, we want to make sure that any changes that occur happen to the expected points in the expected category.

Reply:
Ok that's fine, we want to change how the active category stuff works to be more usable and less clunky anyways we can fix this part of the ux up when we do that.